src/views/forms/StakeForm/index.tsx

```diff
@@ -203,15 +217,22 @@ const $Description = styled.div`
 const $Footer = styled.footer`
   ${formMixins.footer}
   --stickyFooterBackdrop-outsetY: var(--dialog-content-paddingBottom);
+
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
 `;
 const $WithDetailsReceipt = styled(WithDetailsReceipt)`
   --withReceipt-backgroundColor: var(--color-layer-2);
 `;
 
-const $FormInputToggleButton = styled(ToggleButton)`
-  ${formMixins.inputInnerToggleButton}
+const $LegalDisclaimer = styled.div`
+  text-align: center;
+  color: var(--color-text-0);
+  font: var(--font-small-book);
+`;
 
-  svg {
-    color: var(--color-text-0);
-  }
+const $Link = styled(Link)`
+  --link-color: var(--color-text-2);
+  display: inline-flex;
```

Comment:
can we add a cursor: pointer here?

Reply:
hmm doesn't `Link` already have `cursor: pointer` [here](https://github.com/dydxprotocol/v4-web/blob/main/src/components/Link.tsx#L64)? (I see a pointer when I hover over Keplr / Stride as well, but lmk if I'm missing something!)